public/locales/en/common.json

```diff
@@ -134,13 +134,16 @@
       "LESSON_PART": "Daily Lessons",
       "DAILY_LESSON": "Daily Lessons",
       "VIRTUAL_LESSON": "Webinars",
+      "VIRTUAL_LESSONS": "Webinars",
```

Comment:
What's difference between VIRTUAL_LESSON and VIRTUAL_LESSONs?

Reply:
VIRTUAL_LESSONs is collection content type that have same title on filters